client/public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.gif" />
```

Comment:
This should still be `.ico`? The file that was added is an `.ico`.

Reply:
I think I missed a 'git add .' during the final revision of that PR work.

Thanks for the careful eye, Ino. Fixed in this PR.

Not all browsers will support .gif favicons yet without strenuous javascripting so I've included reducededs of both for HTML5 browsers to choose their preference with the tagging pointing to the gif.

Reply:
I don't think animated favicons are good to have, so I'd prefer to just have the `.ico`.